tests/ui/specs/MediaPlayer/MediaPlayer-specs.js

```diff
@@ -0,0 +1,95 @@
+const Page = require('./MediaPlayerPage');
+
+describe('MediaPlayer', function () {
+
+	beforeEach(function () {
+		Page.open();
+	});
+
+	describe('default', function () {
+		const mediaPlayer = Page.components.mediaPlayerDefault;
+
+		it('should have the slider knob focused', function () {
+			expect(mediaPlayer.slider.isFocused()).to.be.true();
+		});
+
+		it('should play media on playButton click', function () {
+			expect(mediaPlayer.knob.getCSSProperty('left').value).to.equal('0px');
+
+			mediaPlayer.playButton.click();
+			Page.delay(1000);
```

Comment:
Very occasionally, we found a failure in the local. This seems to be an error due to the inability to play media.
As Jenkins machine's performance lower than local, I think it needs enough delay to play the media.
failure is as follow:
AssertionError: expected '0px' to not equal '0px'
(/home/enact/agate/agate/tests/ui/specs/MediaPlayer/MediaPlayer-specs.js:22:65)

Reply:
Good idea. Thank you!